### ISSUE TITLE
Waymo two stage question or bug？

### ISSUE THREAD

**Issue by DongfeiJi** [tianweiy/CenterPoint#168]
In  CenterPoint\det3d\models\detectors\single_stage.py has a func called freeze
--------------------
`    def freeze(self):
        for p in self.parameters():
            p.requires_grad = False
        FrozenBatchNorm2d.convert_frozen_batchnorm(self)
        return self`
-------------------
It is for freeze one stage part,
but in PillarFeatureNet, it also has BatchNorm1d, who's running_mean  and running_var  is not freezed
Is this a bug? 
--------------
`single_det.reader.pfn_layers.0.norm.running_mean tensor(0.6021)
single_det.reader.pfn_layers.0.norm.running_var tensor(89.9156)
single_det.reader.pfn_layers.1.norm.running_mean tensor(3.8551)
single_det.reader.pfn_layers.1.norm.running_var tensor(16.7426)`

**Comment by tianweiy** [comment]
yes, that's a bug. I don't think it will matter too much for the performance though.

**Comment by DongfeiJi** [comment]
> yes, that's a bug. I don't think it will matter too much for the performance though.

i try try

**Comment by sjeevanandam** [comment]
Hi,
When I am using a single GPU, the batchnorm weight and bias's requires_grad are properly set to False. But when I use more than one GPU i.e. via DDP the same issue happens. Any ideas?
